samples/algorithms/sudoku-grover/ColoringGroverWithConstraints.qs

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Quantum.Samples.ColoringGroverWithConstraints {
+
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Diagnostics;
+
+    /// # Summary
+    /// Read color from a register
+    operation MeasureColor (register : Qubit[]) : Int {
+        return ResultArrayAsInt(MultiM(register));
+    }
+
+    /// # Summary
+    /// Read coloring from a register
+    operation MeasureColoring (K : Int, register : Qubit[]) : Int[] {
+        let V = Length(register) / K;
+        let colorPartitions = Partitioned(ConstantArray(V-1, K), register);
+        return ForEach(MeasureColor, colorPartitions);
+    } 
+
+    /// # Summary
+    /// N-bit color equality oracle (no extra qubits)
```

Comment:
I'm missing the inputs and outputs section in some of the docstring here, that might be helpful to add.

Reply:
done